### ISSUE TITLE
cannot clap while buckled to a chair

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#29173]
(WEB REPORT BY: applecrocodile REMOTE: 206.221.180.138:7777)
# Revision
7215558089bd24c506ad0844b9a92450daac8ef7

# Description

# Steps to Reproduce
buckle to chair, *clap
# What you Expected
to clap
# What Actually Happened
You cannot clap while restrained.

**Comment by Eneocho** [comment]
Chairs and all shouldn't use cuffedcode unless you were actually cuffed.

**Comment by robintime** [comment]
yeah the way emotes work is they have restraint_check on ones that wouldn't be usable while cuffed or while immobile. this could be fixed by separating restraint check into different checks, like cuffed_check or buckled_check or something like that

**Comment by Inorien** [comment]
addressed in #35040
